angular/core/components/poll/dot_vote/votes_panel_stance/poll_dot_vote_votes_panel_stance.coffee

```diff
@@ -0,0 +1,21 @@
+angular.module('loomioApp').directive 'pollDotVoteVotesPanelStance', (PollService, RecordLoader) ->
+  scope: {stance: '=', poll: '='}
```

Comment:
It looks like poll isn't used in here? Also if we did need I reckon `$scope.stance.poll()` would be the way to do it.